makefile

```diff
@@ -74,23 +75,23 @@ makefile : ;
 
```

Comment:
Remove `+@$(call MAKE_DIR,$(BUILD_PATH)/$(APP))` (creates empty app folder now). Don't remember why this was needed, maybe windows compatibility. Should not be needed with current supported build envs.

Reply:
Line 65. Couldn't comment on that line for some reason

Reply:
I removed the creation of the empty app directory, but left `+@$(call MAKE_DIR,$(BUILD_PATH))` to create the `build` directory.